.github/BOTMETA.yml

```diff
@@ -950,8 +950,9 @@ files:
   $module_utils/jboss/:
     maintainers: $team_jboss
   $module_utils/scaleway.py: &scaleway
-    maintainers: sieben hekonsek
+    maintainers: $team_scaleway
     labels:
+      - scaleway
```

Comment:
The `scaleway` GitHub label doesn't exist yet and needs to be manually created. @resmo could you create it please ?

Reply:
i would like to forward it to the repo owners. /cc @gundalow

Reply:
Created https://github.com/ansible/ansible/labels/scaleway